NUSGrabberGUI/Form1.cs

```diff
@@ -193,49 +199,60 @@ private void STTitleList_SelectedIndexChanged(object sender, EventArgs e)
             {
                 if (v != null && v != "v0" && v != "" && v != " ")
                 {
-                    try { STVersionList.Items.Add(v.TrimStart(' ').Substring(0, v.LastIndexOf('(') - 1)); }

```

Comment:
The exception thrown here was causing the system titles to take a long time to filter.